imports/server/migrations/36-call-history-indexes.ts

```diff
@@ -0,0 +1,10 @@
+import { Migrations } from 'meteor/percolate:migrations';
+import CallHistories from '../../lib/models/mediasoup/call_histories';
+
+Migrations.add({
+  version: 36,
+  name: 'Add indexes to CallHistory collection',
+  up() {
+    CallHistories.createIndex({ call: 1 }, { unique: true });
```

Comment:
Can I request a (non-unique) index on `hunt` as well?  I'm hoping to add some form of this data to the PuzzleListPage.

Totally fine if you prefer I do that in a separate migration later though